content/pages/privacy-policy.md

```diff
@@ -0,0 +1,372 @@
+
+We take complaints very seriously and we treat them as an opportunity to develop our approach. This is why we are always very grateful to hear from people who are willing to take the time to help us improve.
+
+Our policy is:
+
+- To provide a fair complaints procedure that is clear and easy to use for anyone wishing to make a complaint.
+- To publicise the existence of our complaints procedure so that people know how to contact us to make a complaint.
+- To make sure everyone in our organisation knows what to do if a complaint is received.
+- To make sure all complaints are investigated fairly and in a timely way.
+- To make sure that complaints are, wherever possible, resolved and that relationships are repaired.
+- To learn from complaints and feedback to help us to improve what we do.
+
+#### Confidentiality
+
+All complaint information will be handled sensitively, in line with relevant data protection requirements.
+
+#### Responsibility
+
+Overall responsibility for this policy and its implementation lies with the Director of Marketing and Development
+
+For further information on how to make a complaint, please click here.
+
+Information Commissioner's Office
+For further assistance with complaints regarding your data, please contact the Information Commissioner's Office, whose remit covers the UK.
+
+Information Commissioner's Office \
+Wycliffe House \
+Water Lane \
+Wilmslow \
+SK9 5AF
+
+Telephone: 0303 123 1113 \
+Email: [casework@ico.org.uk](mailto:casework@ico.org.uk)
+
+### 8. Our use of cookies
+
+Our website uses cookies, as almost all websites do, to help provide you with the best experience we can. Cookies are small text files that are placed on your computer or mobile phone when you browse websites.
+
+We use cookies, which are issued by third party service providers that we continuously assess to make sure the data is secure. We collect these cookies in order to ensure that anything we do in our online advertising and campaigns are cost effective and relevant. We track how well our adverts perform and then analyse this data to make sure we can continue to make improvements.
+
+Our cookies help us:
+
+- Make our website work as you'd expect
+- Remember your settings during and between visits
+- Improve the speed/security of the site
+- Allow you to share pages with social networks like Facebook
+- Continuously improve our website for you
+
+We do NOT use cookies to:
+
+- Collect any personally identifiable information (without your express permission)
+- Collect any sensitive information (without your express permission)
+- Pass personally identifiable data to third parties
+- Pay sales commissions
+
+You can learn more about all the cookies we use below.
```

Comment:
We need to cut this list down to only the cookies we use (Us - to remember cookie settings, GA & Hotjar)

Reply:
I've removed the cookie providers we don't use and made a new issue to audit the exact cookies we use at the end of development - #132 -  I've not added it to this milestone.